src/components/admin/input-with-character-limit/InputWithCharacterLimit.scss

```
@@ -72,5 +81,24 @@
+            text-align: right;
+            width: 100%;
+        }
+
+        &--visually-hidden {
+            position: absolute;
+            width: 1px;
+            height: 1px;
+            padding: 0;
+            margin: -1px;
+            overflow: hidden;
+            clip: rect(0 0 0 0);
```

Stylelint (17.4.0)

[error] Unexpected deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)